composer.json

```diff
@@ -32,7 +32,7 @@
     "drupal-composer/drupal-scaffold": "^2",
     "drupal/admin_toolbar": "^1.0",
     "drupal/adminimal_admin_toolbar": "^1.9",
-    "drupal/apigee_edge": "1.x-dev",
+    "drupal/apigee_edge": "^1.0",
```

Comment:
I think (from my testing) this causes `1.x-dev` (which technically is the rc-3 tag on drupal.org) to install, no matter what, because there is no official release yet.

```
 - Installing drupal/apigee_edge (dev-1.x ee98c6c): Cloning ee98c6c4ee from cache
```

I think what you actually want is: `"drupal/apigee_edge": "^1.0@rc"`, which gives:

```
- Installing drupal/apigee_edge (1.0.0-rc3): Downloading (100%)
```

---

Either way, the end result is the same, even if we leave it, simply because of the way we do development on Github, and only push to the Drupal repo there when it's time to roll a release.

Reply:
I don't think we want to limit to rc versions by using `"drupal/apigee_edge": "^1.0@rc"`, but instead use ^1.0 so that rc and any 1.0 GA releases are pulled down as they are released.

Using ^1.0 pulls down the closest 1.0 version to an actual GA release, so this seems to be correct, since you can see using this PR gives you `* 1.0.0-rc3` (marked below with *), not 1.x-dev:

```
$ composer show drupal/apigee_edge --all
name     : drupal/apigee_edge
descrip. : Apigee Edge for Drupal.
versions : 1.x-dev, * 1.0.0-rc3, 1.0.0-rc2, 1.0.0-rc1, 1.0.0-beta2, 1.0.0-beta1, 1.0.0-alpha9, 1.0.0-alpha8, 1.0.0-alpha7, 1.0.0-alpha6, 1.0.0-alpha5, 1.0.0-alpha4, 1.0.0-alpha3, 1.0.0-alpha2, 1.0.0-alpha1, dev-1.x
type     : drupal-module
....

```

Reply:
Yeah. I think it’s fine, and the behavior I was seeing (it was giving me 1.x-dev) was because of the way I tested it, without having prefer-stable set to true. We have that in our composer project and here, so it should be ok.